lib/hawkular/operations/operations_api.rb

```diff
@@ -326,12 +326,15 @@ def add_jdbc_driver(hash, &callback)
     # Exports the JDR report
     #
     # @param [String] resource_path canonical path of the WildFly server
+    # @param [Boolean] delete_from_server specifies whether the temporary file at the remote server should be deleted.
     # @param callback [Block] callback that is run after the operation is done
-    def export_jdr(resource_path, &callback)
+    def export_jdr(resource_path, delete_from_server = false, &callback)
       fail Hawkular::ArgumentError, 'resource_path must be specified' if resource_path.nil?
       check_pre_conditions(&callback)
 
-      invoke_specific_operation({ resourcePath: resource_path }, 'ExportJdr', &callback)
+      invoke_specific_operation({ resourcePath: resource_path,
+                                  deleteImmediately: delete_from_server },
```

Comment:
same here, it reads a bit strange to me `deleteImmediately = delete_from_server` (not disputing the fact it works, just the code readability aspect) . it just sounds there's another option "delete_from_somewhere_else"